zuul.d/ansible-network-jobs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+- job:
+    name: network-ee-integration-tests-ansible-network
+    parent: ansible-network-vyos-appliance
+
+- job:
+    name: network-ee-integration-tests-ansible-network
```

Comment:
ha, we should move set collection_namespace / collection_name here along with skip_tag. Then only set libssh below

Reply:
actually, I'm not sure we need skip_tags